test/validate/eia_test.py

```diff
@@ -74,9 +74,9 @@ def test_minmax_rows(pudl_out_eia,
     _ = (
         pudl_out_eia.__getattribute__(df_name)()
         .pipe(pv.check_min_rows, expected_rows=expected_rows,
-              margin=0.05, df_name=df_name)
+              margin=0.0, df_name=df_name)
         .pipe(pv.check_max_rows, expected_rows=expected_rows,
-              margin=0.05, df_name=df_name)
+              margin=0.0, df_name=df_name)
```

Comment:
why turn the margin to 0? that seems extreme given that we know there will be some fluctuation

Reply:
Haha, very true.

Reply:
I think we may often want to know about these consequences though. Like fixing the leading-zeroes in generator IDs does change the number of records in some cases since the aggregations lead to fewer groupings. In this case I also wanted to do it so that I could update the expected number of rows to reflect the actual expected number of rows. It's more a warning that "Hey, something changed! Did you expect it to change? If not, maybe you should look into why it changed."

Reply:
yeah that makes sense to me. especially in the world in which we have the validation tests running every night.

in the before times of the nightly validation tests (now!), I think this behavior is going to be hard to interact with. We will tweak things over time, not realize when and have to play forensic investigator for issues that are probably mostly non-issues.

Reply:
Yeah this is definitely with an 👁️ toward the aftertimes. I guess I was also thinking that since we run the validation tests so rarely now it's not going to be a common annoyance. And we can look at the numbers and be like "Meh, close enough." if we want to.